### ISSUE TITLE
Design of the WebApp

### ISSUE THREAD

**Issue by thogebati** [amosproj/amos2023ws01-ticket-chat-ai#12]
## User story
1. As a user that creates a ticket in the WebApp
2. I want / need a visual pleasing design
3. So that I get the feeling of using a well maintained Tool

## Acceptance criteria
* Only Siemens approved colours are used
* User friendly (intuitive)
* The language is german

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by thogebati** [comment]
To maybe add, that it also looks good on a mobile browser

**Comment by AnnaH3003** [comment]
Ideally, this is what it should look like:

![Image](https://github.com/amosproj/amos2023ws01-ticket-chat-ai/assets/92970997/17cca7e0-6e0c-423f-872c-8a4e8b6cb279)
